task-list-rails/app/controllers/tasks_controller.rb

```diff
@@ -1,4 +1,15 @@
 class TasksController < ApplicationController
   def index
+    @tasks = Task.all
+  end
+
+  def show
+    id = params[:id]
+    @task = Task.find(id)
+    @task_info_hash = @task.attributes
```

Comment:
cool use of the attributes! Seems like you could use the `@task.attributes` directly though without requiring an additional value. You also don't use the `@task` instance variable in your view at all so if you were to only use the `@task_info_hash` the `@task` variable should be a local variable rather than an instance variable.